IdentityCore/src/oauth2/aad_base/MSIDAADTokenResponse.h

```diff
@@ -41,4 +41,7 @@
 // Derived properties
 @property (readonly) NSDate *extendedExpiresOnDate;
 
+- (NSError *)getOAuthError:(id<MSIDRequestContext>)context
```

Comment:
I think this should be in the Token response class.

Reply:
Agree